src/hooks/useIsFeatureEnabled/index.tsx

```diff
@@ -15,6 +15,7 @@ const featureFlags = {
   historyRoute: [ChainId.BSC_MAINNET, ChainId.BSC_TESTNET],
   vaiRoute: [ChainId.BSC_MAINNET, ChainId.BSC_TESTNET],
   createProposal: [ChainId.BSC_MAINNET, ChainId.BSC_TESTNET],
+  voteProposal: [ChainId.BSC_MAINNET, ChainId.BSC_TESTNET],
```

Comment:
I'd add tests to cover the `createProposal` and `voteProposal` feature flags (we currently test all other feature flags).

Reply:
Done!

Reply:
Thank you